packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -417,26 +410,23 @@ export async function makeWallet({
       .then(payoutObj => {
         const payoutIndexToKeyword = [];
         return Promise.all(
-          Object.entries(payoutObj).map(([keyword, payoutP], i) => {
-            // keyword may be an index for zoeKind === 'indexed', but we can still treat it
-            // as the keyword name for looking up purses and payouts (just happens to
-            // be an integer).
-            payoutIndexToKeyword[i] = keyword;
-            return payoutP;
-          }),
-        ).then(payoutArray =>
-          Promise.all(
-            payoutArray.map(async (payoutP, payoutIndex) => {
+          Object.entries(payoutObj)
+            .map(([keyword, payoutP], i) => {
+              // keyword may be an index for zoeKind === 'indexed', but we can still treat it
```

Comment:
This comment is stale, please remove it (I do note that you didn't change this, but while you're here...).

Reply:
Comment removed with general comment.